Packs/Tessian/Integrations/Tessian/Tessian.yml

```diff
@@ -0,0 +1,113 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Tessian
+  version: -1
+configuration:
+- display: Portal URL
+  name: url
+  additionalinfo: The URL that you use to access the Tessian Portal. Please include the extension, e.g. "example.tessian-platform.com" or "example.tessian-app.com"
+  type: 0
+  required: true
+# - display: API Key
+#   name: api_key
+#   additionalinfo: The API Key to use to connect to the Tessian API. This can be found under "Security Integrations" in your Tessian Portal (/0/admin/integrations/api/tokens)
+#   type: 0
+#   required: true
```

Comment:
```suggestion
# - display: API Key
#   name: api_key
#   additionalinfo: The API Key to use to connect to the Tessian API. This can be found under "Security Integrations" in your Tessian Portal (/0/admin/integrations/api/tokens)
#   type: 0
#   required: true
```
remove